docs/source/scripts_and_their_functions.rst

```diff
@@ -26,6 +25,12 @@ User Input Forwarder
 --------------------
 This script mimics a distribution node. It listens to documents coming from the User Input Producer on ``ws://127.0.0.1:9001`` and forwards them to any consumer listening on ``ws://127.0.0.1:9000``. Like the Simple Producer, it can also save the documents it receives to the file system. First, run it with the ``--folder-export`` argument like this: ``ebu-user-input-forwarder --folder-export myFolder``. Then launch the User Input Producer and connect. The sequence will be saved to ``myFolder`` along with the manifest file. The User Input Forwarder can also be used as an incoming connection point for WebSocket connections from sources other than the User Input Producer.
 
+Handover Manager
+----------------
+This node implements the 'Who claimed control most recently' algorithm defined in the specification. This algorithm determines the output from multiple input sequences.
```

Comment:
Sorry, I just realised we ought to come clean and say "defined in the specification, with added functionality to allow messages to be broadcast between members of the authors group, which may be added to a future specification."